### PR TITLE
#13424926 integrate travis ci with readme

### PR DESCRIPTION
**Pull request by victor-enogwe**
#### What does this PR do?
Add Travis ci to check code pushed to branches  so we can discover very quickly if your commit broke something, and fix it before it becomes a problem.
#### Description of Task to be completed?

The integration of Travis ci is required to automatically run tests and other preliminary build steps required to ship the next iteration of the product

#### How should this be manually tested?
N/A

#### Any background context you want to provide?
created a .travis.yml file with travis ci configuration set and slack notifications enabled

#### What are the relevant pivotal tracker stories?
#13424926 Integrate TravisCI with readme badge

#### Screenshots (if appropriate)
[Build Status](https://travis-ci.org/andela/anbu-cfh)

#### Questions:
N/A

### CODE REVIEW

**Comment by andela-aolaniran** [comment]
No screen shot of the travis ci badge.

**Comment by victor-enogwe** [comment]
Thank you andela-olaniran screenshot added

File: .env [hunk 1]
```diff
@@ -0,0 +1 @@
+MONGOHQ_URL=mongodb://localhost/anbu-cfh
```
Comment (neddinn) [inline .env:1]: This file should be added to your .gitignore you don't want to share some sensitive information in the repo.

File: README.md [hunk 2]
```diff
@@ -1,4 +1,5 @@
-[![codebeat badge](https://codebeat.co/badges/aa2e3819-2f12-4593-83ab-10660560550c)](https://codebeat.co/projects/github-com-andela-anbu-cfh)
```
Comment (neddinn) [inline README.md]: Did you pull from staging? This page should not be removed if so.

**Comment by neddinn** [comment]
Your code review template is missing the header `#### What does this PR do?` Please add that.

File: README.md [hunk 3]
```diff
@@ -1,4 +1,5 @@
...
+[![Build Status](https://travis-ci.org/andela/anbu-cfh.svg?branch=chore%2F13424926%2Fintegrat-travis-ci-with-readme)](https://travis-ci.org/andela/anbu-cfh) [![Coverage Status](https://coveralls.io/repos/github/andela/anbu-cfh/badge.svg?branch=master)](https://coveralls.io/github/andela/anbu-cfh?branch=master) [![codebeat badge](https://codebeat.co/badges/aa2e3819-2f12-4593-83ab-10660560550c)](https://codebeat.co/projects/github-com-andela-anbu-cfh)
```
Comment (neddinn) [inline README.md:1]: Why do you have the coveralls badge? Is that part of the scope for your story?

Comment (victor-enogwe) [reply README.md:1]: I pulled from staging

Comment (neddinn) [reply README.md:1]: Sure? Staging has only the codebeat badge though.

**Pull request closed without merging**